multirange.js

```diff
@@ -61,38 +63,41 @@ var multirange = function(input) {
 		ghost.oninput = input.oninput.bind(input);
 	}
 
-	function update() {
+	function update(m) {
 		ghost.style.setProperty("--low", 100 * ((input.valueLow - min) / (max - min)) + 1 + "%");
 		ghost.style.setProperty("--high", 100 * ((input.valueHigh - min) / (max - min)) - 1 + "%");
+		if (drag && m !== 1) {
+			let w = input.valueHigh - input.valueLow;
+			if (w>1) w-=0.5;
+			middle.style.setProperty("--size", (100 * w / (max - min)) + "%");
+			middle.value = min + (input.valueHigh + input.valueLow - 2*min - w)*(max-min)/(2*(max - min - w));
+		}
 	}
 
 	ghost.addEventListener("mousedown", function passClick(evt) {
-		// Are the ghost and input elements inverted? (ghost is lower range)
-		var isInverted = input.valueLow == ghost.value;
-		// Find the horizontal position that was clicked (as a percentage of the element's width)
-		var clickPoint = evt.offsetX / this.offsetWidth;
-		// Map the percentage to a value in the range (note, assumes a min value of 0)
-		var clickValue = max * clickPoint;
-
-		// Get the distance to both high and low values in the range
-		var highDiff = input.valueHigh - clickValue;
-		var lowDiff = Math.abs(input.valueLow - clickValue);
-
-		if (isInverted? highDiff < lowDiff : lowDiff < highDiff) {
-			// The low value is closer to the click point than the high value
-			// We should update the low value input
-			var passEvent = new MouseEvent("mousedown", evt);
-			// Pass a new event to the low "input" element (which is obscured by the
-			// higher "ghost" element, and doesn't get mouse events outside the drag handle
-			input.dispatchEvent(passEvent);
-			// The higher "ghost" element should not respond to this event
-			evt.preventDefault();
-			return false;
+		// Find the horizontal position that was clicked
+		let clickValue = min + (max - min)*evt.offsetX / this.offsetWidth;
+		let middleValue = (input.valueHigh + input.valueLow)/2;
+		if ( (input.valueLow == ghost.value) == (clickValue > middleValue) ) {
+			// Click is closer to input element and we swap thumbs
+			input.value = ghost.value;
```

Comment:
What happened here? It looks like some functionality was removed? (and some comments too)

Reply:
I just solved the same problem in another simpler way. Ghost handle is always moved and where it goes through the middle, another handle is put in the previous place. The old method with click event simulation is not workable in Chrome.

Reply:
This is from the previous PR, I not understand why it appears there